README.md

```diff
@@ -25,7 +25,11 @@ It is built as a part of Microverse Web Developer Course.
 
 ## Getting Started
 
-To get a local copy up and running usual steps.
+To get a local copy up and running usual steps. Because there is no framework or any other special way. There is only plain JS, HTML and CSS. So you can get a local copy and running it in a code editor.
+
+To get the content of this project locally you need to run this command in your terminal:
+
+- `https://github.com/tacettinkustu/microverse-awesome-books.git`

```

Comment:
## [Optional]
Good job so far.

I think you're trying to list some steps for non-technical users to set up for your app locally.

Example: 
- git clone `your repo`
- cd into the folder
- open `index.html` with live server